webui/react/src/components/ColumnPickerMenu.tsx

```diff
@@ -165,26 +197,63 @@ const ColumnPickerTab: React.FC<ColumnTabProps> = ({
   const rows = useCallback(
     ({ index, style }: { index: number; style: React.CSSProperties }) => {
       const col = filteredColumns[index];
+      const colType = col.type.replace('COLUMN_TYPE_', '');
+      const getColDisplayName = (col: ProjectColumn) => {
+        const metCol = metadataColumns.get(col.column);
+
+        if (metCol !== undefined && metCol.length > 1) {
```

Comment:
if we're always including the badge, do we still need the column count state and this branch?